src/Schema/Directives/CountDirective.php

```diff
@@ -39,6 +39,18 @@ public static function definition(): string
   Apply scopes to the underlying query.
   """
   scopes: [String!]
+
+  """
+  The field to count.
+  counts only a specific field
+  """
+  field: String
+
+  """
+  distinct count.
+  exclude duplicated rows
+  """
+  distinct: Boolean
```

Comment:
The second argument **field** below will take the column name, this is only to exclude duplicate rows when counting records.

Reply:
Got it, makes sense to have this as a `Boolean` then. The definition can be cleaned up though, since passing `null` here does not make sense and there is a clear default:

```suggestion
  """
  Should exclude duplicated rows?
  """
  distinct: Boolean! = false
```